kotest-property/src/commonMain/kotlin/io/kotest/property/Gen.kt

```diff
@@ -1,7 +1,5 @@
 package io.kotest.property
 
-import io.kotest.fp.NonEmptyList
-import io.kotest.fp.Option
```

Comment:
Should we also remove both Option and NonEmptyList?

Reply:
If they're otherwise unused yes I will.

Reply:
Option is used elsewhere, NEL is gone.